config/index.js

```diff
@@ -0,0 +1,35 @@
+// see http://vuejs-templates.github.io/webpack for documentation.
+let path = require("path");
+
+module.exports = {
+	build: {
+		env: { NODE_ENV: "\"production\"" },
+		index: path.resolve(__dirname, "../../api/src/Template/Layout/layout_a.ctp"),
+		assetsRoot: path.resolve(__dirname, "../../api/webroot/"),
```

Comment:
What are these paths?

Reply:
oups sorry ! I based my work on an existing project. I will fix that ASAP